integrations/influxdb_connector/influxdb-timestream-connector/template.yml

```diff
@@ -0,0 +1,591 @@
+              },
+              "stageVariables": {
+                #foreach($key in $stageVariables.keySet())
+                "$key": "$util.escapeJavaScript($stageVariables.get($key))" #if($foreach.hasNext),#end
+                #end
+              },
+              "isBase64Encoded": false
+            }
+          text/csv: |
+            {
+              "resource": "$context.resourcePath",
+              "path": "$context.path",
+              "httpMethod": "$context.httpMethod",
+              "body" : "$util.escapeJavaScript($input.body)",
+              "headers": {
+                #foreach($param in $input.params().header.keySet())
+                "$param": "$util.escapeJavaScript($input.params().header.get($param))" #if($foreach.hasNext),#end
+                #end
+              },
+              "queryStringParameters": {
+                #foreach($param in $input.params().querystring.keySet())
+                "$param": "$util.escapeJavaScript($input.params().querystring.get($param))" #if($foreach.hasNext),#end
+                #end
+              },
+              "pathParameters": {
+                #foreach($param in $input.params().path.keySet())
+                "$param": "$util.escapeJavaScript($input.params().path.get($param))" #if($foreach.hasNext),#end
+                #end
+              },
+              "requestContext": {
+                "accountId": "$context.identity.accountId",
+                "resourceId": "$context.resourceId",
+                "stage": "$context.stage",
+                "requestId": "$context.requestId",
+                "requestTimeEpoch": "$context.requestTimeEpoch",
+                "identity": {
+                  "cognitoIdentityPoolId": "$context.identity.cognitoIdentityPoolId",
+                  "accountId": "$context.identity.accountId",
+                  "cognitoIdentityId": "$context.identity.cognitoIdentityId",
+                  "caller": "$context.identity.caller",
+                  "apiKey": "$context.apiKey",
+                  "sourceIp": "$context.identity.sourceIp",
+                  "cognitoAuthenticationType": "$context.identity.cognitoAuthenticationType",
+                  "cognitoAuthenticationProvider": "$context.identity.cognitoAuthenticationProvider",
+                  "userArn": "$context.identity.userArn",
+                  "userAgent": "$context.identity.userAgent",
+                  "user": "$context.identity.user"
+                },
+                "resourcePath": "$context.resourcePath",
+                "httpMethod": "$context.httpMethod",
+                "apiId": "$context.apiId"
+              },
+              "stageVariables": {
+                #foreach($key in $stageVariables.keySet())
+                "$key": "$util.escapeJavaScript($stageVariables.get($key))" #if($foreach.hasNext),#end
+                #end
+              },
+              "isBase64Encoded": false
+            }
+          application/json: |
```

Comment:
I would like to simply have some kind of variable I could use so that all of these mapping templates aren't repeated but I don't believe such a thing exists for SAM templates.

Why there are so many mapping templates is that the InfluxDB Go client actually sends `application/json` requests, while the official documentation says to use `text/plain; charset=utf-8`, so multiple content types must be supported.